src/parameterizations/lateral/MOM_lateral_mixing_coeffs.F90

```diff
@@ -1089,10 +1294,27 @@ subroutine VarMix_init(Time, G, GV, US, param_file, diag, CS)
        'Inverse eddy time-scale, S*N, at u-points', 's-1', conversion=US%s_to_T)
     CS%id_SN_v = register_diag_field('ocean_model', 'SN_v', diag%axesCv1, Time, &
        'Inverse eddy time-scale, S*N, at v-points', 's-1', conversion=US%s_to_T)
-    call get_param(param_file, mdl, "VARMIX_KTOP", CS%VarMix_Ktop, &
-                 "The layer number at which to start vertical integration "//&
-                 "of S*N for purposes of finding the Eady growth rate.", &
-                 units="nondim", default=2)
+    call get_param(param_file, mdl, "USE_NEW_EADY_GROWTH_RATE", CS%use_new_Eady_growth_rate, &
```

Comment:
"New" is an adjective that might be best avoided in a name, since something that is new now won't be new in a few months or years.  What would you think about calling this something like "SIMPLER_EADY_GROWTH_RATE" instead?

Reply:
What if we come up with an even simpler approach? 😁  `SIMPLER_...` it is.

Reply:
Parameter name changed in d21d21acc78308143836df2ddaf22f99e6485a52